### DISCUSSION TITLE
What does the * mean, on the drives overview

### DISCUSSION THREAD

**Discussion by jakobfriis** [teslamate-org/teslamate#4140] [General]
I noticed the * and wondered what it means. Any thoughts ?

![image](https://github.com/user-attachments/assets/37d232c6-b0fd-40f6-a827-496819c35bce)

**Comment by jakobfriis** [comment]
From the code, i can see that it relates to "reduced_range" but where is that data coming from, or how is that calculated ?

**Reply by cwanja** [reply]
If I recall, there is a logic that determines if your battery is "cold" and that range would be impacted. If the useable SOC differs than the SOC at start - meaning the battery could be "cold soaked". There used to be a tooltip when you moused over it, guessing that was deprecated with a Grafana update. 

Believe this is the logic:
`case when (start_position.battery_level != start_position.usable_battery_level OR end_position.battery_level != end_position.usable_battery_level) = true then true else false end  as reduced_range,`

See #257 and #2255
